polynote-frontend/polynote/cell.js

```diff
@@ -86,6 +86,8 @@ export class Cell extends UIEventTarget {
         super(id, content, language);
         this.id = id;
         this.language = language;
+        if (!language) throw {message: `Attempted to create cell ${id} with empty language!`};
+        this.isCode = false;
```

Comment:
Just curious – you use `isCode` in one place; why not just `cell instanceof CodeCell`?

Reply:
🤦‍♂️